subsys/usb/usb_device.c

```diff
@@ -1368,6 +1387,18 @@ int usb_transfer_sync(u8_t ep, u8_t *data, size_t dlen, unsigned int flags)
 	return pdata.tsize;
 }
 
+int usb_wakeup_request(void)
```

Comment:
Who calls it, class driver, user app?

Reply:
User application.

Reply:
Split into 2 commits; extended hid_mouse example.